examples/meta/src/evaluation/cross_validation.sg

```diff
@@ -0,0 +1,91 @@
+#Cross Validation for Binary Classifier
+File f_feats_train = csv_file("../../data/classifier_binary_2d_linear_features_train.dat")
+File f_feats_test = csv_file("../../data/classifier_binary_2d_linear_features_test.dat")
+File f_labels_train = csv_file("../../data/classifier_binary_2d_linear_labels_train.dat")
+File f_labels_test = csv_file("../../data/classifier_binary_2d_linear_labels_test.dat")
+Math:init_random(2)
+
+#![create_features]
+Features features_train = features(f_feats_train)
+BinaryLabels labels_train(f_labels_train)
+Labels label_train_new = labels(f_labels_train)
+Features features_test = features(f_feats_test)
+BinaryLabels labels_test(f_labels_test)
+#![create_features]
+
+#![set_parameters]
+real C = 1.0
+real epsilon = 0.001
+#![set_parameters]
+
+#![create_instance]
+Machine svm = machine("LibLinear", labels=label_train_new, epsilon=epsilon, C1=C, liblinear_solver_type=enum LIBLINEAR_SOLVER_TYPE.L2R_L2LOSS_SVC)
+#![create_instance]
+
+#![create_cross_validation]
+StratifiedCrossValidationSplitting splitting_strategy(labels_train, 2)
```

Comment:
could you patch `factory.h` to have the cross-validation splitting as part of it? We wanna use the new API everywhere if possible

Reply:
This is a good one indeed, but for the sake of going ahead with this PR, I suggest to do that independently. Are we keeping track of the stuff that is still missing factories somewhere?